content/docs/start/metrics-parameters-plots.md

```diff
@@ -0,0 +1,257 @@
+---
+title: 'Get Started: Metrics, Parameters, and Plots'
+---
+
+# Get Started: Metrics, Parameters, and Plots
+
+DVC makes it easy to track [metrics](/doc/command-reference/metrics), update
+[parameters](/doc/command-reference/params), and visualize performance with
+[plots](/doc/command-reference/plots). The tutorial below introduces these
```

Comment:
Minor: not sure we want to call this a "tutorial". I'm probably biased against that because we used to have a whole Tutorials section with longer walk-throughs (only https://dvc.org/doc/use-cases/versioning-data-and-model-files/tutorial is left, and a bunch of unofficial ones in 3rd party blogs).

Reply:
(Same in the next line)

Reply:
Maybe "We introduce these features below, and the Experiments page shows..."

> Note also I avoided term "concepts" since we're working on a "Concepts" section of the guide — lots of terminology concerns sorry 😆

Reply:
Thoughts on the updates?

Reply:
Checking updates now!